File: medicfy/lib/src/models/frequency.dart
class Frequency {
  int frequencyId;
  String type;
  int daysNumber;
  Frequency({this.frequencyId, this.daysNumber, this.type});
  Map<String, dynamic> toMap() {
    Map<String, dynamic> map = new Map();
    map['frequencyId'] = frequencyId;
    map['type'] = type;
    map['daysNumber'] = daysNumber;
    return map;
  }
}

File: BackEndMedicFy/Models/Frequency.cs
using System.ComponentModel.DataAnnotations;
namespace BackEndMedicFy.Models
{
    public class Frequency
    {
        [Key]
        public int FrequencyId { get; set; }
        public string Type { get; set; }
        public int DaysNumber { get; set; }
    }
}